api/envoy/config/filter/http/aws_lambda/v2alpha/aws_lambda.proto

```diff
@@ -12,7 +12,7 @@ option java_outer_classname = "AwsLambdaProto";
 option java_multiple_files = true;
 option (udpa.annotations.file_migrate).move_to_package =
     "envoy.extensions.filters.http.aws_lambda.v3";
-option (udpa.annotations.file_status).work_in_progress = true;
+option (udpa.annotations.file_status).work_in_progress = false;
```

Comment:
Wouldn't this be the default?

Reply:
I think this should remain as `true`.  If we're making this `false` we may as well remove this option line.